apps/vs-code-designer/src/app/commands/createNewProject/createProjectSteps/ScriptProjectCreateStep.ts

```diff
@@ -73,8 +79,15 @@ obj
 appsettings.json
 local.settings.json
 __blobstorage__
+.debug
+.gitignore
```

Comment:
Committing or not ignoring the .gitignore, as far as I know is a good practice. I don't know if we want to ignore  it but, lets think about the person that clones the project, the .gitignore will not be cloned.